GroupProject/Pages/Index.cshtml

```diff
@@ -7,12 +7,12 @@
 
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
```

Comment:
Deleted this section because it was the standard text and not something specific for the webpage